skabase/SKABaseDevice/SKABaseDevice/SKABaseDevice.py

```diff
@@ -220,85 +228,107 @@ def get_device_attributes(self, with_value=False,
 
```

Comment:
The new device properties added in the XML file are not present in the Python file.  Was the file regenerated using POGO?

Reply:
Yes it was regenerated using Pogo earlier. Not needed now.